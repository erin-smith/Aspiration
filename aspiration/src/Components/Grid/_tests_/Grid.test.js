import * as React from "react";
import {Col, Row, Container} from "../index";
import renderer from "react-test-renderer";

describe("Grid", () => {
  test("Col should match the snapshot", () => {
    const tree = renderer.create(<Col size="md-12"/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("Row should match the snapshot", () => {
    const tree = renderer.create(<Row/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("Container should match the snapshot", () => {
    const tree = renderer.create(<Container/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
