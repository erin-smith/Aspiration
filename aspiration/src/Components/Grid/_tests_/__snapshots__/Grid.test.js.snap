// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Col should match the snapshot 1`] = `
<div
  className="col-md-12"
/>
`;

exports[`Grid Container should match the snapshot 1`] = `
<div
  className="container"
/>
`;

exports[`Grid Row should match the snapshot 1`] = `
<div
  className="row"
/>
`;
